x-pack/plugins/infra/public/pages/metrics/inventory_view/lib/color_from_value.ts

```diff
@@ -79,7 +79,7 @@ export const calculateSteppedGradientColor = (
       return rule.color;
     }
     return color;
-  }, (first(rules) as any).color || defaultColor);
+  }, first(rules)!.color || defaultColor);
```

Comment:
Could this be expressed as

```suggestion
  }, first(rules)?.color ?? defaultColor);
```

to avoid the assertion?